python/nbextension/src/chemiscope-bootstrap.less

```diff
@@ -1,3 +1,15 @@
 .chemiscope-bootstrap {
     @import (less) 'bootstrap/dist/css/bootstrap.css';
 }
+
+.fade {
```

Comment:
Unfortunately that's the point. The `.chemiscope-bootstrap` wrapper is downstream of `.modal`, and Jupyter causes `.modal` to have opacity 0. The only solution I see is to overwrite Jupyter's styles.

Reply:
No I meant having `chemiscope-fade` and then moving this CSS to the main css (not only in notebooks) & changing the HTML to use the new classes.

Reply:
I'm not sure I understand, but the problem is that `fade` is set by Bootstrap, not in the HTML.

Reply:
we set `fade` manually here https://github.com/lab-cosmo/chemiscope/blob/f74c88dd7a760a6e74437c39ff7066c3a5726e99/src/map/options.html#L1. Or you are talking about another thing? 

Anyway, this is something we can change later when using our own JS for modal opening/closing right? Then I'll be fine with temporarily overwriting jupyter CSS.

Reply:
No you're right actually, but we still need to keep `fade` becauses Bootstrap needs it:
https://github.com/twbs/bootstrap/blob/273ebad151b98c37ebedc5111c3efff596dd0d24/js/src/modal.js#L36-L44

But then I suppose we can add a `chemiscope-modal` class to the div to avoid overwriting, while keeping the `modal`/`fade`/`show` classes used by the JavaScript (perhaps that's what you meant all along).

Reply:
Something like this (I was a bit unclear on the details, but if it is not too much work I would rather be a nice player in the ecosystem & not touch external CSS)